src/routes/EditHoldingRoute.test.js

```diff
@@ -5,6 +5,10 @@ import { render, screen } from '@folio/jest-config-stripes/testing-library/react
 
 import EditHoldingRoute from './EditHoldingRoute';
 
+jest.mock('@folio/service-interaction', () => ({
+  NumberGeneratorModalButton: () => <div>NumberGeneratorModalButton</div>
+}));
```

Comment:
You could mock it in one [place](https://github.com/folio-org/ui-inventory/tree/master/test/jest/__mock__)

Reply:
With the `transformIgnorePatterns` the mock is not necessary. I removed them.